stores/facilitiesStore.ts

```diff
@@ -14,9 +14,22 @@ export const useFacilitiesStore = defineStore(
             facilityData.value = facilityResults
         }
 
+        async function updateFacility(facility: MutationUpdateFacilityArgs) {
+            try {
+                return await graphQLClientRequestWithRetry(
+                    gqlClient.request.bind(gqlClient),
```

Comment:
Future PR: Is there a way we don't have to pass in a binding function? It's not very intuitive.